cryptocurrency/script.py

```diff
@@ -0,0 +1,54 @@
+from deephaven import jpy
+import os
+os.system("pip install pycoingecko") #install the package to use
+from pycoingecko import CoinGeckoAPI
+from deephaven.TableTools import merge
+from time import sleep
+from deephaven.DBTimeUtils import convertDateTime, secondsToTime, millisToTime
+import threading
+import time
+
+timeToWatch = 2 # minutes to keep table updating
+secondsToSleep = 10 # should be 10 or higher. If too fast will hit request limit
+
+getHistory = True
+daysHistory = 90 # if getHistory = true the days to pull
+
+#coins to get data on so far
+ids=['bitcoin', 'ethereum','litecoin', 'dogecoin', 'tether', 'binancecoin','cardano', 'ripple', 'polkadot']
+
+#tables to merge current and previous data
+tableArray=[]
+
+DynamicTableWriter = jpy.get_type("io.deephaven.db.v2.utils.DynamicTableWriter")#org.joda.time.base.BaseDateTime works
+tableWriter = DynamicTableWriter(["Coins", "DateTime", "prices", "market_caps", "total_volumes"], [jpy.get_type("java.lang.String"), jpy.get_type("io.deephaven.db.tables.utils.DBDateTime"), jpy.get_type("double"), jpy.get_type("double"), jpy.get_type("double")])
+
+tableArray.append(tableWriter.getTable())
+
+cg = CoinGeckoAPI()
+
+#for historical data
+if getHistory:
+    for names in ids:
+        coin_data_hist =cg.get_coin_market_chart_by_id(names, vs_currency="usd", days=daysHistory)
+        sub = pd.DataFrame(coin_data_hist)
+        tableArray.append(dataFrameToTable(sub).update("Coins = names").update("DateTime= millisToTime((long)prices_[i][0])", "prices = prices_[i][1]", "market_caps = market_caps_[i][1]", "total_volumes = total_volumes_[i][1]").moveUpColumns("Coins","DateTime"))
+
+result = merge(tableArray).selectDistinct("DateTime", "Coins", "prices", "market_caps", "total_volumes").sortDescending("DateTime") #add each coin data to the master table
```

Comment:
Er, where is `result` used? Where does it go after it's computed here?

Reply:
Added to readme! It is a table in the IDE